meta/classes/nfsroot.bbclass

```diff
@@ -1,38 +1,36 @@
 python nfsroot_eventhandler() {
-    import socket
-    import os
+    # prevent self-calling
```

Comment:
Did you intentionally remove the comment here?

    # NFS_UPDATER_INTERNAL is set by nfs-root-updater to prevent calling itself